src/web/theme/modules/Tile/ContentTile/ContentTile.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./ContentTile.scss";
+
+const ContentTile = ({ title, children, style }) => {
+  const contentClass = `content-tile${style ? "--" + style : ""}`;
```

Comment:
Told you this in an earlier PR, your element should **always** have the base `content-title` class, and you should add the full modifier when needed.

in your DOM this would look like this :
`<div class="content-title">` for the base title
`<div class="content-title content-title--modifier">` when a modifier is applied

This would simplify your below css rules a lot

Reply:
aa :disappointed: that's true I forgot :/ my bad I think it's fixing the adrien's commentary